zebra-chain/src/history_tree.rs

```diff
@@ -239,8 +239,8 @@ impl NonEmptyHistoryTree {
 
     /// Extend the history tree with the given blocks.
     pub fn try_extend<
-        'a,
-        T: IntoIterator<Item = (Arc<Block>, &'a sapling::tree::Root, &'a orchard::tree::Root)>,
+        'b,
+        T: IntoIterator<Item = (Arc<Block>, &'b sapling::tree::Root, &'b orchard::tree::Root)>,
```

Comment:
Nitpick: This change looks unnecessary, and I'd initially thought there might be another lifetime `'a` in the impl block, so it could be clearer to leave it as `'a`.

```suggestion
        'a,
        T: IntoIterator<Item = (Arc<Block>, &'a sapling::tree::Root, &'a orchard::tree::Root)>,
```

Reply:
change made (sorry cant quite remember why i did this)